### ISSUE TITLE
Intra-class compactness

### ISSUE THREAD

**Issue by mnikitin** [pengyuLPY/Virtual-Fully-Connected-Layer#6]
Hello! 
I've read the paper and your approach really looks very promising for resources-limited FR training!

My question is whether Virtual FC itself forces in any way intra-class compactness during training?
From my perspective, inter-group anchors and regrouping strategy both improve inter-class separability, but not intra-class compactness.
As far as I understand, multi-task learning helps with it.
Is it the only solution proposed in the paper, or I'm missing something?

And one more question, what's the idea behind sampling several images per identity in a mini-batch?
It looks like Virtual FC doesn't require it.

Thank you!

**Comment by pengyuLPY** [comment]
Thank you for your interest in our work. Sampling several images per identity in a mini-batch is for two motivations: 1. estimation the corresponding anchor precisely. 2. the intra-class compactness. 

Motivation 1: The corresponding anchor for ID_i is estimated by multiple images of ID_i in the mini-batch. We think multiple sampling images can improve the precision of the anchor estimation (i.e., the centroid of ID_i representation). The influence of sampling numbers is shown in Tab.2

Motivation 2: The corresponding anchor of ID_i works as the weight in the final Cls FC layer. There are multiple images of ID_i sampled in the mini-batch. The softmax-like loss function (e.g., ArcFace loss) will compact the intra-class representations to the anchor (centroid).

I hope your questions are answered. Just let me know if there are any concerns left ^_^
